pkg/ccl/backupccl/file_sst_sink.go

```diff
@@ -303,6 +312,11 @@ func (s *fileSSTSink) copyPointKeys(dataSST []byte) error {
 	defer iter.Close()
 
 	for iter.SeekGE(storage.MVCCKey{Key: keys.MinKey}); ; iter.Next() {
+		if err := s.pacer.Pace(ctx); err != nil {
```

Comment:
How expensive is a `Pace()` call? Should we consider adding an int and checking this only 1/1000 loop iterations?, We could also add an option to Writer to call it during flush, or add a shim to the io.Writer, so we'd only check pacing during flushes (i.e. during block writes)?